Sourcery/Parsing/Parser.swift

```diff
@@ -375,6 +375,12 @@ final class Parser {
 
         for (_, type) in unique {
             if let enumeration = type as? Enum, enumeration.rawType == nil {
+                enumeration.cases.forEach { enumCase in
+                    enumCase.associatedValues.forEach { associatedValue in
+                        associatedValue.type = unique[associatedValue.typeName]
```

Comment:
you should resolve typealiases here, you can copy the snippet that variable uses just above this for loop, (I'll be refactoring those repetitions separately).